tomcat/src/main/java/org/apache/coyote/http11/request/RequestMapping.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.coyote.http11.request;
+
+import org.apache.coyote.http11.controller.DefaultController;
+import org.apache.coyote.http11.controller.LoginController;
+import org.apache.coyote.http11.controller.RegisterController;
+import org.apache.coyote.http11.controller.RootController;
+import org.apache.coyote.http11.Controller;
+
+import java.util.Map;
+
+import static org.apache.coyote.http11.response.ResponsePage.LOGIN_PAGE;
+import static org.apache.coyote.http11.response.ResponsePage.REGISTER_PAGE;
+import static org.apache.coyote.http11.response.ResponsePage.ROOT_PAGE;
+
+public class RequestMapping {
```

Comment:
객체 이름이 행위여서 조금 어색한 것 같아요!! 😊
메서드가 행위에 해당할테니 객체 네이밍은 해당 행위를 하는 주체를 표현해보는 건 어떨까요??

```suggestion
public class RequestControllerMapper {
```